daemon/pod/persist.go

```diff
@@ -410,14 +410,14 @@ func (v *Volume) removeFromDB() error {
 	return removeMessage(v.p.factory.db, fmt.Sprintf(VX_KEY_FMT, v.p.Id(), v.spec.Name), v, "volume info")
 }
 
-func (inf *Interface) saveInterface() error {
+func (inf *Interface) saveInterface(id string) error {
```

Comment:
why is the id different with the `Id` field in the data structure? should we correct the data structure instead of pass a new id here?

Reply:
@gnawux the reason is inf.descript.Id is not set anywhere in hyperd.

Reply:
@gnawux I made a mistake with this part.
Please see the comments of https://github.com/hyperhq/hyperd/pull/697/commits/a5ac58d322dc58ed52713dfcd3b4359d39c87348 about this.